_pages/mobile-interview-project.md

```diff
@@ -182,6 +182,10 @@ Definitely! Please utilize as many Google searches and Stack Overflow searches a
 
 That being said, please do not bulk copy and paste large sections of code from 3rd party sources, as this would defeat the purpose of the evaluation. If you do copy in code, please note as such in your README.
 
+ * Q) **Can I utilize AI code generation tools like GitHub Copilot?**
+
+You may not use AI code generation tools because we want this project to be a reflection of _your_ skills as a developer. We also do not allow the use of GitHub Copilot internally at Block due to IP/legal reasons.
```

Comment:
I don't think this is true, or at least, I haven't found any written policy that explicitly forbids it.

Reply:
removed in latest!